user_mgr.cpp

```diff
@@ -258,8 +258,7 @@ void UserMgr::throwForMaxGrpUserCount(
     }
     else
     {
-        if (usersList.size() > 0 && (usersList.size() - getIpmiUsersCount()) >=
-                                        (maxSystemUsers - ipmiMaxUsers))
+        if (usersList.size() > 0 && (usersList.size() >= maxSystemUsers))
         {
             log<level::ERR>("Non-ipmi User limit reached");
```

Comment:
Given this now includes both IPMI and non-IPMI users, this should be changed to simply "User limit reached".
(Same update to the log message 3 lines below)